config/rbac/rbac.yaml

```diff
@@ -60,3 +60,90 @@ rules:
       - watch
       - get
       - list
+
+# These ClusterRoles are used to provide aggregated permissions to the
+# built-in Kubernetes default aggregated roles: admin, edit, view
+# See https://kubernetes.io/docs/reference/access-authn-authz/rbac/#user-facing-roles
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: cartographer-user-admin
+  labels:
+    rbac.authorization.k8s.io/aggregate-to-admin: "true"
+rules:
+  - apiGroups:
+      - carto.run
+    resources:
+      - clustersupplychains
+      - clustersourcetemplates
+      - clusterimagetemplates
+      - clustertemplates
+      - clusterdeliveries
```

Comment:
What does a ClusterRole referencing cluster scoped resources do when applied via a RoleBinding in a namespace? My expectation is that the cluster scoped resources are ignored, but want to verify that. We don't want to give namespace admins access to cluster scoped resources.

Reply:
```
$ kubectl get workloads
NAME      AGE
sockeye   2d10h
$ kubectl get clustersupplychains
Error from server (Forbidden): clustersupplychains.carto.run is forbidden: User "evana@vmware.com" cannot list resource "clustersupplychains" in API group "carto.run" at the cluster scope
```

This is with a rolebinding of `edit` in the `default` namespace.

Reply:
It does seem from [the k8s docs](https://kubernetes.io/docs/reference/access-authn-authz/rbac/#user-facing-roles) and [this CNCF blog post](https://www.cncf.io/blog/2020/08/28/kubernetes-rbac-101-authorization/) that these roles are intended to be used _only_ for namespaced resources. Therefore, I think I'd be in favor of dropping the cluster scoped resources from all of these roles.

(This would make the admin and edit roles identical, so maybe we could use one ClusterRole with both labels for aggregation.)

Reply:
Done.